app/views/shared/_nav-list.html.erb

```diff
@@ -3,10 +3,10 @@
     <li><%= link_to("My Demands", me_path, class: is_active_path(me_path)) %></li>
     <li><%= link_to("New Demand", new_demand_path, class: is_active_path(new_demand_path)) %></li>
   <% end %>
-  <% if user_area.present? %>
-    <li><%= link_to(user_area.zip_codes.first.city, "#") %></li>
-    <li><%= link_to("#{user_area.zip_codes.first.county}", "#") %></li>
-    <li><%= link_to(user_area.zip_codes.first.state, "#") %></li>
+  <% if user_municipality.present? %>
+    <li><%= link_to(user_municipality.short_name, search_url) %></li>
+    <li><%= link_to(user_municipality.county.short_name, URI.encode("#{search_url}?scope=county")) %></li>
```

Comment:
No need to `URI.encode()` here, `link_to` takes keys/values for query params after the URL so like:

```
link_to(user_municipality.county.short_name, search_url, scope: 'county')
```

should work.